backend/lcfs/web/api/compliance_report/views.py

```diff
@@ -119,9 +118,7 @@ async def get_compliance_report_summary(
     Retrieve the comprehensive compliance report summary for a specific report by ID.
     """
     await validate.validate_organization_access(report_id)
-    return await summary_service.calculate_compliance_report_summary(
-        report_id, request.user
-    )
+    return await summary_service.calculate_compliance_report_summary(report_id)
```

Comment:
Without passing in the user, how do we exclude draft summary calculations for government users? Gov users should not see summary calculations from a supplemental draft report changes. Are you handling this at the schedule level now?

Reply:
And once we go into the world of re-assessment, that will no longer be true as Government users will be the ones creating the draft records